### DISCUSSION TITLE
How I can create MIDI?

### DISCUSSION THREAD

**Discussion by Kir2406** [onlaj/Piano-LED-Visualizer#289] [Q&A]
I am a musician and have no understanding of programming. But really want to record tracks with led! 

Can I record the midi piece I'm playing directly in the RPI? And then download from microSD to your computer? Or should I connect rpi to PC? And how to do it? Only via Bluetooth / WiFi or is it possible via a wire? 

Sorry for my stupid questions🤦‍♂️

**Comment by onlaj** [comment]
> Can I record the midi piece I'm playing directly in the RPI?

Yes.

> And then download from microSD to your computer?

This, or you can use [web interface](https://github.com/onlaj/Piano-LED-Visualizer#web-interface)

**Reply by Kir2406** [reply]
Thank you🙏

**Reply by Kir2406** [reply]
I still didn't understand :(

Where is MIDI recorded on a microsd? When I insert the microsd into my computer, there are no recorded midis. But RPI sees them and can record and play them. 
How can I get these records without a WEB interface?

**Reply by onlaj** [reply]
It's located under /home/Piano-LED-Visualizer/Songs
As long as your RPi is connected to WiFi you can also use sftp or ssh to transfer files, it's much easier than switching sd card.

**Reply by Kir2406** [reply]
Thank you so much!

But what is this path? Where is /home/Piano-LED-Visualizer/Songs?.. 
I open microsd and only see OS files. 1 folder "overlays" and many separate files

**Reply by onlaj** [reply]
It seems that on Windows those files are not accessible by default https://raspberrypi.stackexchange.com/questions/12806/read-a-file-from-the-raspberry-pis-sd-card-with-windows

**Reply by Kir2406** [reply]
Thank you! 🙏
You are my savior! 😁
